src/Components/ImagesTable/ImagesTableToolbar.tsx

```diff
@@ -0,0 +1,114 @@
+import React, { useState } from 'react';
+
+import {
+  Pagination,
+  Toolbar,
+  ToolbarContent,
+  ToolbarItem,
+  Title,
+} from '@patternfly/react-core';
+import { Link } from 'react-router-dom';
+
+import {
+  selectSelectedBlueprintId,
+  selectBlueprintSearchInput,
+} from '../../store/BlueprintSlice';
+import { useAppSelector } from '../../store/hooks';
+import {
+  BlueprintItem,
+  useGetBlueprintsQuery,
+} from '../../store/imageBuilderApi';
+import { resolveRelPath } from '../../Utilities/path';
+import { useExperimentalFlag } from '../../Utilities/useExperimentalFlag';
+import { BlueprintActionsMenu } from '../Blueprints/BlueprintActionsMenu';
+import { BuildImagesButton } from '../Blueprints/BuildImagesButton';
+import { DeleteBlueprintModal } from '../Blueprints/DeleteBlueprintModal';
+
+interface imagesTableToolbarProps {
+  itemCount: number;
+  perPage: number;
+  page: number;
+  onSetPage: (event: React.MouseEvent, page: number) => void;
+  onPerPageSelect: (event: React.MouseEvent, perPage: number) => void;
+}
+
+const ImagesTableToolbar: React.FC<imagesTableToolbarProps> = ({
+  itemCount,
+  perPage,
+  page,
+  onSetPage,
+  onPerPageSelect,
+}: imagesTableToolbarProps) => {
+  const experimentalFlag = useExperimentalFlag();
```

Comment:
I think we might avoid bugs more if this header component is _only_ used for the Blueprints table. Then we can just check the experimentalFlag one time, in the <Table> component, and render either the old header or this new header component conditionally.

Reply:
It's done now ;)